test-data/unit/check-incremental.test

```diff
@@ -4206,3 +4206,14 @@ class Two:
     pass
 [out2]
 tmp/m/two.py:2: error: Revealed type is 'builtins.str'
+
+[case testImportUnusedIgnore]
```

Comment:
Does it make sense to also add the second test case from #2960?

Reply:
Not sure why the second case went missing, but:
It turns out that it fails because of another bug, which I have filed (#4856)